protocol-dashboard/src/store/cache/analytics/hooks.ts

```diff
@@ -200,6 +206,28 @@ async function fetchTimeSeries(
   return metric
 }
 
+async function fetchUptime(node: string, bucket: Bucket) {
+  if (bucket !== Bucket.DAY) {
+    // currently only 24h uptime supported
```

Comment:
it would be a lot of keys to loop through for 7d or 30d, but just fyi the /uptime endpoint accepts any arbitrary number via the `durationHours` query param